prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'Andrew Owyang';
+      console.log(fullName);
+      console.log(typeof fullName);
+      var numberOfPets = 0;
+      console.log(numberOfPets);
+      console.log(typeof numberOfPets);
+      var likesVegetables = true;
+      console.log(likesVegetables);
+      console.log(typeof likesVegetables);
+      var empty = null;
+      console.log(empty);
+      console.log(typeof empty);
+      var unset;
+      console.log(unset);
+      console.log(typeof unset);
+    </script>
```

Comment:
Rather than using `var`, we want to use `const` or `let`. `var` is an outdated way of creating variables, and moving forward you will usually only use `const` or `let`.